src/sdk/trace/sampler.rs

```diff
@@ -5,11 +5,11 @@ use crate::api;
 #[derive(Clone, Debug)]
 pub enum Sampler {
     /// Always sample the trace
-    Always,
+    AlwaysOn,
     /// Never sample the trace
-    Never,
+    AlwaysOff,
     /// Sample if the parent span is sampled
-    Parent,
+    ParentOrElse,
```

Comment:
From looking at the spec this should be a composite sampler, so likely something like `ParentOrElse(Box<Sampler>)`, and the logic should reflect the inner sampler when there is no parent.

Reply:
Thank you for your comment. Sorry, yes, I reread the spec and I now think I am following it 😄  Please let me know if my changes are on the right track. Thanks!